examples/tanstack-start/app.config.ts

```diff
@@ -4,7 +4,12 @@ import tsConfigPaths from "vite-tsconfig-paths";
 export default defineConfig({
 	vite: {
 		plugins: [
-			postgresPlugin(),
+			postgresPlugin({
+				seed: {
+					type: "sql-script",
+					path: "./schema.sql",
+				},
```

Comment:
If people copy-paste this the `referrer` would be unknown.
Let's put a value like `github:neondatabase/neondb-cli/examples/tanstack-start/app.config.ts`, and add a comment telling them they should change this ASAP with their project's name:
```js
// IMPORTANT: This helps us understand where DB creations on Neon come from.
// Please change this as soon as you can, to add your project name:
// "github:org/your-repo" or "npm:your-package"
```